src/pull.js

```diff
@@ -0,0 +1,94 @@
+/* @flow */
+
+import printit from 'printit'
+
+import RemoteCozy from './remote/cozy'
+import Prep from './prep'
+import Pouch from './pouch'
+import * as conversion from './conversion'
+
+import type { RemoteDoc } from './remote/document'
+import type { LocalDoc } from './local/document'
+
+const log = printit({
+  prefix: 'Pull',
+  date: true
+})
+
+// Pull changed files and directories metadata from the remote Cozy, and merge
+// them into the local database.
+export default class Pull {
```

Comment:
I'm not sure either.
Right now it only pulls the metadata, delegating part of the work to the `Prep` class.

I was first thinking about making some "pull" domain notion visible by extracting the corresponding logic out of `RemoteWatcher`, including the filesystem update.
Then I figured out that it was actually only updating metadata in pouch, and that including fs update in the same class may actually not be such a good idea.

I assumed I could wait to understand the fs part better, and eventually rename it to something else.